### ISSUE TITLE
Text Span to Link BUG

### ISSUE THREAD

**Issue by jamescrabb** [soflyy/oxygen-bugs-and-features#778]
**Text links within a paragraph:**

Tried selecting the portion of text I want to be a link and wrapping in a span. Now select the link button and the span text disappears! 

**I have replicated this on the sandbox environment with the same results.**

I have discovered that if you use the “set” button to give the link wrapper created a target link the text reappears but it will be above the text almost as though it has been ejected from the text element. 

Strange bug but certainly annoying. Has anyone else experienced this?

You can replicate it by:
1. create section 
2. Add text element 
3. Highlight a portion of the text
4. Make selection a span 
5. With the span selected make it a link
6. Voila it will vanish like magic
7. Select the 'Set' button in the link wrapper and the text link appears outside the paragraph it was created in (also removes the text that was highlighted)

There are some simple work arounds like using rich text element and writing the css yourself but this is a bug that should not be occurring.

**Comment by twistedpixel** [comment]
Yep, this is extremely frustrating. I'm pretty sure it's a Safari issue (because Safari is a total PoS these days). In fact I think it actually got worse as I now can't select anything in a span without the entire span's content disappearing.
